8.x/theme-development/helpers.md

```diff
@@ -10,424 +10,1608 @@ First, open the helper file: `app/Utils/CMS/Template/helpers.php`.
+__________________________
+
 
 
-### get_latest_blog()
+### get_latest_blog
+:::tip
+function get_latest_blog($type, $count)
+:::
 Returns the latest blog posts with that specified number. The type object will be removed in later versions, and this function will only receive the count object from the input.
+__________________________
 
 
-### get_suggested_blog()
+### get_suggested_blog
+:::tip
+function get_suggested_blog($type, $count)
+:::
 Returns suggested_blog with that specified number. The type object will be removed in later versions, and this function will only receive the count object from the input.
+__________________________
 
 
-### get_system_messages()
+### get_system_messages
+:::tip
+function get_system_messages()
+:::
 Sometimes an error occurs when sending data. In order for the system to send a message and be able to receive the message, this function must be placed in the footer of the website.
+__________________________
 
-### has_system_messages()
+### has_system_messages
+:::tip
+function has_system_messages(): bool
+:::
 Checks if the system has a message.
+__________________________
 
 
-### get_months()
+### get_months
+
 Returns the list of months.
+__________________________
+
+
+### get_years
 
-### get_years()
 Returns the list of years used in the form.
+__________________________
+
+
+
+### hide_number
 
-### hide_number()
 Using this function, you can hide part of a number and use the star character instead.
+__________________________
+
+
+
+### hide_text
 
-### hide_text()
 Using this function, you can hide part of a text and use the star character instead.
+__________________________
+
+
+
+### get_payment_drivers
 
-### get_payment_drivers()
 This function is to display the Larammerce project supporter logo. Currently, the Hinza logo is displayed on Larammerce projects.
+__________________________
+
+
 
+### is_default_payment_driver
 
-### is_default_payment_driver()
 Returns active payment gateways.
+__________________________
+
+
+
+### get_disabled_setting_appliances
 
-### get_disabled_setting_appliances()
 This function returns appliances that are inactive.
+__________________________
+
+
 
+### is_selected
 
-### is_selected()
 Takes the directory from the input and checks that it is in url part. This function will be deprecated in the next version.
+__________________________
+
+
+
+### get_configurations
 
-### get_configurations()
 Returns configurations in the env file.
+__________________________
+
+
 
+### get_searched_products
 
-### get_searched_products()
 Searches the product based on the given query.
+__________________________
+
+
 
+### get_digits
 
-### get_digits()
 Returns the list of digits based on the given language.
+__________________________
 
 
-### convert_digits()
+
+### convert_digits
+
 Converts digits from English to Persian.
```

Comment:
Converts digits from English to Persian. -> Converts digits from english to persian.